### PR TITLE
Bump version to 1.0.0

### PR DESCRIPTION
**Pull request by Sergeykot**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by Sergeykot** [comment]
@lagartoflojo comments addressed. please check again.
I would also need my post-install message to be checked

File: package/rmt-server.spec [hunk 1]
```diff
@@ -225,6 +226,7 @@ getent passwd %{rmt_user} >/dev/null || \
 %service_add_post rmt-server.target rmt-server.service rmt-server-migration.service rmt-server-mirror.service rmt-server-sync.service
 cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails secrets:setup >/dev/null
 cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails runner -e production "Rails::Secrets.write({'production' => {'secret_key_base' => SecureRandom.hex(64)}}.to_yaml)"
+echo "The installation completed. Please run yast2-rmt to configure the package properly" >> /dev/stdout
```
Comment (Sergeykot) [inline package/rmt-server.spec]: please check the content of the message

Comment (ikapelyukhin) [reply package/rmt-server.spec]: @Sergeykot could you also please fix #165 while you are at it? (I think you just need to redirect `Rails::Secrets.write` line to `/dev/null`)

Comment (Sergeykot) [reply package/rmt-server.spec]: done

File: package/rmt-server.spec [hunk 2]
```diff
@@ -224,9 +217,9 @@ getent passwd %{rmt_user} >/dev/null || \
 
 %post
 %service_add_post rmt-server.target rmt-server.service rmt-server-migration.service rmt-server-mirror.service rmt-server-sync.service
-cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails secrets:setup >/dev/null
-cd /usr/share/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails runner -e production "Rails::Secrets.write({'production' => {'secret_key_base' => SecureRandom.hex(64)}}.to_yaml)"
-echo "The installation completed. Please run yast2-rmt to configure the package properly" >> /dev/stdout
+cd %{_datadir}/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails secrets:setup >/dev/null
+cd %{_datadir}/rmt && runuser -u %{rmt_user} -g %{rmt_group} -- bin/rails runner -e production "Rails::Secrets.write({'production' => {'secret_key_base' => SecureRandom.hex(64)}}.to_yaml)" >/dev/null
+echo "Please run the YaST RMT module (or yast2-rmt from the command line) to complete the configuration of your rmt-server" >> /dev/stdout
```
Comment (lagartoflojo) [inline package/rmt-server.spec]: I believe the command is `yast2 rmt` (no dash) but maybe `yast2-rmt` also works? :man_shrugging:

Comment (lagartoflojo) [reply package/rmt-server.spec]: also, I would say "...of your RMT server", or even just "of your RMT"

"RMT server" reads like "repository mirroring tool server" which is funky haha

Comment (Sergeykot) [reply package/rmt-server.spec]: message updated

**Pull request merged**